docs/self-hosting.md

```diff
@@ -30,6 +30,39 @@ $ docker run renovate/renovate:13
 
 If you wish to configure Renovate using a `config.js` file then map it to `/usr/src/app/config.js` using Docker volumes.
 
+#### Kubernetes
+
+Renovate's official Docker image is compatible with Kubernetes. Here is an example manifest of running Renovate against a GitHub Enterprise server:
+
+```yaml
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: renovate
+spec:
+  schedule: "@hourly"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - name: renovate
+            image: renovate/renovate@13.153.0 # Update this to the latest available and then enable Renovate on the manifest
+# Environment Variables
+            env:
+            - name: RENOVATE_PLATFORM
+              value: "github"
+            - name: RENOVATE_ENDPOINT
+              value: "https://github.company.com/api/v3"
+            - name: RENOVATE_TOKEN
+              value: "your-github-enterprise-renovate-user-token"
+            - name: GITHUB_COM_TOKEN
+              value: "any-personal-user-token-for-github-com-for-fetching-changelogs"
+            - name: RENOVATE_AUTODISCOVER
+              value: "true"
+          restartPolicy: OnFailure
```

Comment:
I'd recommend not setting a restart policy here, but instead setting the CronJob to retry the whole job, because failures will be clearer. Setting `concurrencyPolicy: Forbid` on the CronJob is also a good idea.